backend/src/main/java/com/google/sps/eventsender/EventSender.java

```diff
@@ -0,0 +1,166 @@
+   *     match
+   * @param participant2 Instance of a participant class representing a user in a successful match
+   * @return An array of EventAttendees with both of the participants emails pre set.
+   */
+  public EventAttendee[] createParticipantsArray(
+      com.google.sps.data.Participant participant1, com.google.sps.data.Participant participant2) {
+    return new EventAttendee[] {
+      new EventAttendee().setEmail(participant1.getUsername() + "@google.com"),
+      new EventAttendee().setEmail(participant2.getUsername() + "@google.com"),
+    };
+  }
+
+  /**
+   * Creates a notification trigger within the event so that the user will receive an email reminder
+   * in 5 minutes as wel as a push notification to their screen in 1 minute
+   *
+   * @return Returns these options of notifications to be passed into the event as parameters.
+   */
+  public Event.Reminders setReminderNotifications() {
+    EventReminder[] eventReminders = {
+      new EventReminder().setMethod("email").setMinutes(5),
+      new EventReminder().setMethod("popup").setMinutes(1)
+    };
+    return new Event.Reminders().setUseDefault(false).setOverrides(Arrays.asList(eventReminders));
+  }
+
+  /**
+   * TODO(): Currently assumes that the users will be within the same timezone.
+   *
+   * <p>Main method of this API, creates an Ad-lib event ideally the moment that a match is found.
+   * Upon receiving the two participants as well as the meetingStart and meetingEndTime as well as
+   * their time zone configures the event as well as ads hangout link.
+   *
+   * @param participant1 First Participant that has been matched to someone
+   * @param participant2 Second Participant that has been matched to someone
+   * @param meetingStartTime Time in which the meeting is designated to start at in the format
+   *     "yyyy-mm-dd" *
+   * @param meetingEndTime Time in which the meeting is designated to end at in the format *
+   *     "yyyy-mm-dd"
+   * @param timezone Timezone that they plan to meet in.
+   * @return
+   */
+  public Event createEvent(
+      com.google.sps.data.Participant participant1,
+      com.google.sps.data.Participant participant2,
+      DateTime meetingStartTime,
+      DateTime meetingEndTime,
+      String timezone) {
+    return new Event()
+        .setSummary("Your Ad-lib Session")
+        .setDescription("A chance to spontaneously chat with a fellow Googler.")
+        .setStart(setMeetingStartTime(meetingStartTime, timezone))
+        .setEnd(setMeetingEndTime(meetingEndTime, timezone))
+        .setReminders(setReminderNotifications())
+        .setConferenceData(createVideoMeeting())
+        .setAttendees(Arrays.asList(createParticipantsArray(participant1, participant2)));
+  }
+
+  /**
+   * Configures an Hangouts chat Meeting to be added to our Event.
+   *
+   * @return tHe ConferenceData Object
+   */
+  public ConferenceData createVideoMeeting() {
+    ConferenceData conferenceData = new ConferenceData();
```

Comment:
This can move down into the return

`return new ConferenceData().setEntryPoints( ....) ....`